osu.Game/Overlays/WikiOverlay.cs

```diff
@@ -100,7 +101,16 @@ private void onPathChanged(ValueChangedEvent<string> e)
             cancellationToken?.Cancel();
             request?.Cancel();
 
-            request = new GetWikiRequest(e.NewValue);
+            string[] values = e.NewValue.Split('/', 2);
+
+            if (values.Length > 1 && LanguageExtensions.TryParseCultureCode(values[0], out _))
+            {
+                request = new GetWikiRequest(values[1], values[0]);
```

Comment:
I feel like changing `GetWikiRequest` to take in a `Language` rather than `string` and then use the `out` argument here might sit better?

```suggestion
            if (values.Length > 1 && LanguageExtensions.TryParseCultureCode(values[0], out var language))
            {
                request = new GetWikiRequest(values[1], language);
```

Reply:
I made the suggested change.